client/src/components/QA/Question.jsx

```diff
@@ -13,7 +14,9 @@ class Question extends React.Component {
     };
   }
 
-  addAnswerClicked(id) {
+  addAnswerClicked(e) {
+    ClickedTracker('add Answer');
```

Comment:
How does click tracker work in this instance?

Reply:
It's not finish yet. I still thinking how I implement to my widget.